src/utils/mtev_watchdog.c

```diff
@@ -611,21 +661,49 @@ int update_retries(int* offset, time_t times[]) {
   return 1;
 }
 
-static int watchdog_tick(eventer_t e, int mask, void *unused, struct timeval *now) {
-  it_ticks();
+mtev_watchdog_t *mtev_watchdog_create() {
+  int i;
+  for(i=0; i<MAX_HEARTS; i++) {
+    mtev_watchdog_t *lifeline = &mmap_lifelines[i];
+    if(lifeline->active == HEART_ACTIVE_OFF) {
+      lifeline->active = HEART_ACTIVE_ON;
```

Comment:
The current implementation is on when used.  It make sense to me that if you went through the effort of creating a watchdog, that you wanted to use it.